models/intermediate/int_iterable__list_user_unnest.sql

```diff
@@ -1,27 +1,63 @@
-{{ config( materialized='table') }}
--- materializing as a table because the computations here are fairly complex
+{{ config(
+        materialized='incremental',
+        unique_key='unique_key',
+        incremental_strategy='insert_overwrite' if target.type in ('bigquery', 'spark', 'databricks') else 'delete+insert',
+        file_format='delta',
+        on_schema_change='fail'
+    ) 
+}}
```

Comment:
This is the first time we are using non merge incremental strategy. Can you share how you tested this across the various warehouses to ensure the incremental strategy holds and is consistent across the board?

Reply:
I was under the impression that Postgres and Redshift could not perform the delete+insert strategy. Is that not true on the recent releases of dbt-core?

Reply:
In addition to the above, it does not look like we are testing incremental strategies in the run_models.sh config (we are doing full refreshes for each new run and therefore this incremental strategy is not run during our integration tests). Can we update that to ensure we are testing on each new commit.

Reply:
postgres and redshift can't do `insert_overwrite`  or `merge`  - https://github.com/dbt-labs/dbt-core/blob/main/tests/functional/materializations/test_incremental.py#L45-L52. i think delete+insert is the _only_ strategy they can do. i adapted this config a bit from [amplitude](https://github.com/fivetran/dbt_amplitude/blob/main/models/amplitude__event_enhanced.sql#L6)

and will share validation shortly -- i chose this strategy because of what Garegin said about `merge` on bigquery being inconsistent (so kinda assumed we were not going to use `merge` moving forward?)

Reply:
added the buildkite steps -- but yeah maybe we should wait until you close out T-443408

Reply:
Yup I agree that not using merge going forward is the best approach. Once we have comfort on the incremental strategy then we should be good to possibly consider this approach going forward.

However, I want to make sure we are doing this correctly. Ah yes though you are right that we did shift to using `delete+insert` for redshift and postgres! 👍